quickstart/101-ai-studio/dependent.tf

```diff
@@ -0,0 +1,67 @@
+
+resource "azurerm_resource_group" "default" {
+  name     = "azapi-template-rg-${var.names}"
```

Comment:
Why are we prefacing he name with "azapi-template-"? Shouldn't this just be "rg-"?

Reply:
changed